subsys/usb/usbc/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_USBC_STACK usbc_timer.c usbc_stack.c usbc_tc.c usbc_prl.c usbc_pe.c)
+
+if(CONFIG_USBC_STACK)
+  zephyr_include_directories(${ZEPHYR_BASE}/subsys/usbc)
```

Comment:
why can't headers be placed under `include/` if they are public?

Reply:
I've removed the include part